rotkehlchen/tests/api/test_exchanges.py

```diff
@@ -855,12 +867,29 @@ def test_edit_exchange_account(rotkehlchen_api_server_with_exchanges):
     assert kraken.name == 'my_kraken'
     assert kraken.account_type == DEFAULT_KRAKEN_ACCOUNT_TYPE
 
+    # check that queryranges were succesfuly updated and the others were unmodified
+    expected_ranges_tuple = (Timestamp(0), Timestamp(9999))
+    assert db.get_used_query_range('kraken_trades_mockkraken') is None
+    assert db.get_used_query_range('kraken_margins_mockkraken') is None
+    assert db.get_used_query_range('kraken_asset_movements_mockkraken') is None
+    assert db.get_used_query_range('kraken_trades_my_kraken') == expected_ranges_tuple
+    assert db.get_used_query_range('kraken_margins_my_kraken') == expected_ranges_tuple
+    assert db.get_used_query_range('kraken_asset_movements_my_kraken') == expected_ranges_tuple
```

Comment:
you also put this in the start: `'{str(Location.KRAKEN)}_margins_kraken_boi` so would be nice to assert it's still here now.